client/src/components/Donation/patreon-button.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  donationUrls,
+  patreonDefaultPledgeAmount
+} from '../../../../config/donation-settings';
+import envData from '../../../../config/env.json';
+import PatreonLogo from '../../assets/images/components/patreon-logo';
+
+const { patreonClientId }: { patreonClientId: string | null } = envData as {
+  patreonClientId: string | null;
+};
+
+const PatreonButton = (): JSX.Element | null => {
+  if (
+    !patreonClientId ||
+    !patreonDefaultPledgeAmount ||
+    !donationUrls.successUrl
+  ) {
+    return null;
+  }
+
+  const clientId = `&client_id=${patreonClientId}`;
+  const pledgeLevel = `$&min_cents=${String(patreonDefaultPledgeAmount)}`;
```

Comment:
```suggestion
  const pledgeLevel = `$&min_cents=${patreonDefaultPledgeAmount}`;
```
TS doesn't seem to care.